lib/player.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Player do
+  defstruct [:marker]
+
+  def get_move(marker, board) do
+    get_valid_move(marker, board)
+  end
+
+  def get_valid_move(marker, board) do
+    get_cell_location(marker)
+    |> Validator.is_valid_move(board)
+    |> is_valid_location(marker, board)
+  end
+
+  def is_valid_location({true, move}, _marker, _board) do
```

Comment:
I think this naming is a little confusing. At first glance, it looks like the Player is checking if the location is valid. But then I realized that it is actually doing other things. Maybe something you could use is guard clauses? Have a moment to think about this, and maybe we can chat if it gets harder or confusing.

Reply:
Maybe instead of this function it is possible to use a guard clause?